.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+**Description:** 
+
+**JIRA:** 
+https://edlyio.atlassian.net/browse/EDLY-****
+
+**Testing instructions:**
+
+1. Open page A
+2. Do thing B
+3. Expect C to happen
+4. If D happend instead - check failed.
+
+**Merge checklist:**
+
+- [ ] All reviewers approved
+- [ ] CI build is green
```

Comment:
Right  now there is no CI for Edly. We need to work on this. Please create a new ticket in backlog to add CI for edly edx-platform.

Reply:
Right!